add_up.rb

```diff
@@ -0,0 +1,11 @@
+def add_up(i)
+    sum = 0
+    for item in 1..i do
+        sum = sum + item
+    end
+    return sum
+end
+
+puts add_up(2)
+puts add_up(3)
+puts add_up(4)
```

Comment:
I thought the program was supposed to ask the user for the numbers, not being put manually in the code. This code is alright, but I understand the question is not very clear. I would do this way (totally optional).

![Screenshot from 2020-08-28 11-24-47](https://user-images.githubusercontent.com/47392623/91603023-1bee0e00-e921-11ea-9cf4-a984a6e2866f.png)